doc/make.jl

```diff
@@ -314,6 +314,20 @@ function Documenter.deploy_folder(::BuildBotConfig; devurl, repo, branch, kwargs
     end
     return Documenter.DeployDecision(; all_ok=false)
 end
+function Documenter.post_status(::BuildBotConfig; type, repo::String, subfolder=nothing, kwargs...)
+    if type == "pending"
+        @info "Documentation build in progress"
+    elseif type == "success"
+        @info "Documentation build succeeded"
+    elseif type == "error"
+        error("Documentation build errored")
+    elseif type == "failure"
+        error("Documentation build failed")
+    else
+        error("unsupported status type: $type")
+    end
```

Comment:
Why `"pending"` and `"failure"` too? As far as I can tell, we only call `post_status` with either `"success"` or `"error"`?

Reply:
I was going off 
https://github.com/JuliaDocs/Documenter.jl/blob/8d75805af40466cd8cfa75bce52629fb7f5aa8da/src/deployconfig.jl#L478-L491

Reply:
Ah, fair enough, I missed that.